src/minimega/vm_cli.go

```diff
@@ -303,7 +295,7 @@ name, and a JSON string, and returns the JSON encoded response. For example:
 		HelpLong: `
 Take a screenshot of the framebuffer of a running VM. The screenshot is saved
 in PNG format as "screenshot.png" in the VM's runtime directory (by default
-/tmp/minimega/<vm id>/screenshot.png).
+/tmp/minimega/<vm name>/screenshot.png).
```

Comment:
A few questions about paths:
* What path to use if there is no active namespace? `/tmp/minimega/<name>/...`?
And if so, what to do about a host without a namespace that has the same name as an active namespace? E.g. collision between `/tmp/minimega/<namespace>/...` and `/tmp/minimega/<name>/...` where `<namespace>==<name>`
* Other VM files are in `/tmp/minimega/<vm id>/...`, do we now break this convention for the screenshots?